src/frontend/parallelization_phase.nit

```diff
@@ -95,11 +117,24 @@ end
 		assert nullreturn isa AExpr
 		mainfun.n_block.as(ABlockExpr).n_expr.add(nullreturn)
 
+
 		# Create new body for the annotated fun
-		var s_newbody ="""
+		var s_newbody : nullable String = null
+		if nmethdef.n_signature.n_params.length != 0 then
+			var params = ""
+			for param in nmethdef.n_signature.n_params do
+				params += """{{{param.n_id.text}}},"""
```

Comment:
Instead of concatenating Strings one by one and using a substring at the end to remove the extra `,`, you could have inserted them in an array and have the array joined with `,`, that would be a bit faster at runtime.